pandas/conftest.py

```diff
@@ -1063,6 +1063,17 @@ def any_nullable_int_dtype(request):
     return request.param
 
 
+@pytest.fixture(params=tm.FLOAT_EA_DTYPES)
+def any_nullable_float_dtype(request):
```

Comment:
there is already a float_ea_dtype fixture at L982

Reply:
Indeed, I already added one in the original PR. 

In addition, we actually also have a `dtype` fixture that is only available in the `pandas/tests/arrays/floating` directory that can be used in the test below

Reply:
Ok, I can remove this in favor of the other fixture for now.

In general though do we want to try to be consistent in terms of naming / location given that there's already an `any_nullable_int_dtype` fixture in the base level `conftest.py` (maybe that means not having this latter fixture)?